tests/unit/Commands_test.cpp

```diff
@@ -1,7 +1,4 @@
 /**
- * @file tests/commands_test.cpp
```

Comment:
I removed these two lines (here and elsewhere) as they lead to copy/paste errors and become stale when the file is renamed.